### PR TITLE
Adds docstrings

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
This should _only_ add docstrings. Have a look for style and patterns and I can tweak :) 

r? @remi-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: src/Stripe.net/Entities/Accounts/Account.cs [hunk 1]
```diff
@@ -7,67 +7,148 @@ namespace Stripe
...
 
+        /// <summary>
+        /// Business information about the account.
+        /// </summary>
         [JsonProperty("business_profile")]
         public AccountBusinessProfile BusinessProfile { get; set; }
 
+        /// <summary>
+        /// The business type.
+        /// Possible values:
```
Comment (remi-stripe) [inline src/Stripe.net/Entities/Accounts/Account.cs:30]: Can we make sure to keep all values on just one line? That's what we do in java already https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Account.java#L32

We should just do exactly the same thing and pattern with `One of ...`

File: src/Stripe.net/Entities/Accounts/Account.cs [hunk 2]
```diff
@@ -7,67 +7,148 @@ namespace Stripe
...
+        ///  <c>government_entity</c>
+        ///  <c>individual</c>
+        ///  <c>non_profit</c>
+        /// </summary>
         [JsonProperty("business_type")]
         public string BusinessType { get; set; }
 
+        /// <summary>
+        /// TODO: Field description missing from spec.
+        /// </summary>
```
Comment (remi-stripe) [inline src/Stripe.net/Entities/Accounts/Account.cs:41]: Let's not commit this but just keep this as a solution in your docs generation to easily go and track those down

**Comment by cjavilla-stripe** [comment]
Removed TODO comments and tweaked the logic for adding periods to comments. Our requirements for .net are stronger than those for java. 

ptal @remi-stripe

File: src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs [hunk 3]
```diff
@@ -4,6 +4,11 @@ namespace Stripe
 
     public class AccountCapabilitiesAuBecsDebitPaymentsOptions : INestedOptions
     {
+        /// <summary>
+        /// Passing true requests the capability for the account, if it is not already requested. A
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs:8]: I am guessing we can't detect this should be `<c>true</c>` right? Or we need to fix the docstring to be have the ` around true internally?

Comment (cjavilla-stripe) [reply src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs:8]: Correct.

File: src/Stripe.net/Entities/Accounts/AccountRequirementsError.cs [hunk 4]
```diff
@@ -4,12 +4,24 @@ namespace Stripe
...
...cument_failed_copy</c>, <c>verification_document_failed_greyscale</c>, <c>verification_document_failed_other</c>, <c>verification_document_failed_test_mode</c>, <c>verification_document_fraudulent</c>, <c>verification_document_id_number_mismatch</c>, <c>verification_document_id_number_missing</c>, <c>verification_document_incomplete</c>, <c>verification_document_invalid</c>, <c>verification_document_manipulated</c>, <c>verification_document_missing_back</c>, <c>verification_document_missing_front</c>, <c>verification_document_name_mismatch</c>, <c>verification_document_name_missing</c>, <c>verification_document_nationality_mismatch</c>, <c>verification_document_not_readable</c>, <c>verification_document_not_uploaded</c>, <c>verification_document_photo_mismatch</c>, <c>verification_document_too_large</c>, <c>verification_document_type_not_supported</c>, <c>verification_failed_address_match</c>, <c>verification_failed_business_iec_number</c>, <c>verification_failed_document_match</c>, <c>verification_failed_id_number_match</c>, <c>verification_failed_keyed_identity</c>, <c>verification_failed_keyed_match</c>, <c>verification_failed_name_match</c>, or <c>verification_failed_other</c>.
```
[the remark this answers is not in the document]
Comment (cjavilla-stripe) [reply src/Stripe.net/Entities/Accounts/AccountRequirementsError.cs:9]: Not sure why this line isn't breaking, going to investigate now.

[the remark this answers is not in the document]
Comment (cjavilla-stripe) [reply src/Stripe.net/Entities/Accounts/AccountRequirementsError.cs]: fixed.

File: src/Stripe.net/Entities/Accounts/AccountCompanyVerificationDocument.cs [hunk 5]
```diff
@@ -7,13 +7,27 @@ public class AccountCompanyVerificationDocument : StripeEntity<AccountCompanyVer
     {
         #region Expandable Back
 
+        /// <summary>
+        /// (ID of the File)
```
[the remark this answers is not in the document]
Comment (cjavilla-stripe) [reply src/Stripe.net/Entities/Accounts/AccountCompanyVerificationDocument.cs:11]: Sorry, I can't find my notes about what we wanted to change this to. Do you have an example pattern I could use as reference to generate? 


This is how it looks in visual studios as is:

image


image

[the remark this answers is not in the document]
Comment (remi-stripe) [reply src/Stripe.net/Entities/Accounts/AccountCompanyVerificationDocument.cs:11]: yeah we didn't really write docs, we just discussed it. My point was that it was clearer to say something like "if expanded". But let's punt and keep what you have, it's really easy to change later and at least yours will be consistent across the entire library

**Pull request merged**